### PR TITLE
Fix formatting for updatedDateFields

### PR DESCRIPTION
**Pull request by rollbrettler**

### CODE REVIEW

File: lib/format/status.js [hunk 1]
```diff
@@ -32,17 +32,19 @@ function formatStatusUpdate( data ) {
...
 
       var updateType = statusUpdate[ data.field ]
 
       event.changes.push({
         code: data.field,
         field: updateType && updateType.field,
-        originalValue: data.originalText || data.originalDateUtc,
-        value: data.newText || data.newDateUtc,
+        originalValue: data.originalText || data.originalDateLocal,
+        value: data.newText || data.newDateLocal,
```
Comment (jhermsmeier) [inline lib/format/status.js]: Switching from UTC datetimes to local ones here looks like a breaking change – is this intentional?

Comment (rollbrettler) [reply lib/format/status.js]: For the updatedDateFields I could not find a reference for utc dates? see https://api.flightstats.com/flex/flightstatus/samples/v2/FlightStatus_single_flight_includeDeltas.json

Any idea why it was there in the first place. Maybe I missed something!?

Comment (rollbrettler) [reply lib/format/status.js]: I reverted the change, the actual response includes both, so lets keep the utc one.

Comment (jhermsmeier) [reply lib/format/status.js]: Oh, UTC dates are enabled by the `includeNewFields` extended option in requests:

https://github.com/jhermsmeier/node-flightstats/blob/b13e1178cd6ac04fbc93753c1c4db895f0f21925/lib/flightstats.js#L208-L213

Which is why they don't show up in the examples etc; but there's documentation on them here: https://developer.flightstats.com/api-docs/flightstatus/v2/flightstatusresponse – unforunately `updatedDateFields` does not sport UTC times (yet?)

**Comment by jhermsmeier** [comment]
Published as flightstats@0.20.0

**Pull request merged**